library/js/fancybox/jquery.fancybox-1.2.6.js

```diff
@@ -33,7 +33,7 @@
 	};
 
 	var elem, opts, busy = false, imagePreloader = new Image, loadingTimer, loadingFrame = 1, imageRegExp = /\.(jpg|gif|png|bmp|jpeg)(.*)?$/i;
-	var ieQuirks = null, IE6 = $.browser.msie && $.browser.version.substr(0,1) == 6 && !window.XMLHttpRequest, oldIE = IE6 || ($.browser.msie && $.browser.version.substr(0,1) == 7);
+	var ieQuirks = null, IE6 = false, oldIE = false;
```

Comment:
1.2.6 was the last  version of Fancybox before rewriting to 1.3.x and the 1.x branch was abandoned in 2010.
So yes, we need to upgrade it sometime.